### ISSUE TITLE
GDPR/Privacy

### ISSUE THREAD

**Issue by DeniseColbert** [FreeUKGen/FreeUKGenealogyWP#174]

**Comment by ghost** [comment]
@TazHinkle @johnshaw1000 @DeniseColbert In the effort to move pages from the craft web-site to the development/staging area, there is a page for the cookie policy. I have marked that as being handled separately. I think Taz was working on it and I think John had gotten the page information for him. So, I am marking that page as being handled, separate from the task for the craft web-site:  https://github.com/FreeUKGen/FreeUKGenealogyWP/issues/144#issue-798212400

**Comment by DeniseColbert** [comment]
That's right, @hotbeverage2017 John has created a few 'holding' pages and Taz, too (I can think of the tank you page that the donate tool send users to, from the Bank Transfer/Cheque route). This might be the only one that John has brought the content across for, or there may be others. Worth noting on the spreadsheet, thanks for noticing Gloria :)

**Comment by johnshaw1000** [comment]
Generally the pages I created were the ones in the footer as I didn't want missing/broken links.
